pkg/popeye.go

```diff
@@ -478,6 +486,37 @@ func dumpDir() string {
 	return filepath.Join(os.TempDir(), "popeye")
 }
 
+func parseBucket(bucketURI string) (string, string, error) {
```

Comment:
Should probably add a test for this function to make sure the parsing produces what we want.